src/routes/Player/OptionsMenu/Option/Option.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2017-2022 Smart code 203358507
+
+const React = require('react');
+const PropTypes = require('prop-types');
+const classnames = require('classnames');
+const Icon = require('@stremio/stremio-icons/dom');
+const { Button } = require('stremio/common');
+const styles = require('./styles');
+
+const Option = ({ icon, label, deviceId, disabled, onClick }) => {
```

Comment:
- [ ] We should add some docs for parameters, e.g.:
  - `onClick` should take `deviceId` parameter
  - `label` should be a translated string as the component doesn't translates it
  - etc.